CHANGELOG.md

```diff
@@ -27,6 +27,8 @@ For experimental package changes, see the [experimental CHANGELOG](experimental/
 ### :books: (Refine Doc)
 
 * refactor(examples): use new exported string constants for semconv in examples/http [#4750](https://github.com/open-telemetry/opentelemetry-js/pull/4750) @Zen-cronic
+* refactor(examples): added usage of @opentelemetry/semantic-conventions and @opentelemetry/resources to the examples in examples/opentelemetry-web for maintaining consistency across all examples. [#4764](https://github.com/open-telemetry/opentelemetry-js/pull/4764) @Zen-cronic
+
```

Comment:
nit: This extra line is resulting in lint breaking, `npm run lint:markdown`.  The current markdown rules don't like multiple-blank lines.

Reply:
Gotcha, extra line deleted.